application/spring-boot-app/build.gradle

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'org.springframework.boot' version '2.3.2.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+    id 'java'
+}
+
+group 'org.mobilitydata'
+version '1.3.0-SNAPSHOT_web'
+
+bootJar {
+    // Change the JAR name from spring-boot-app to gtfs-validator
+    archiveBaseName = rootProject.name
+    // Remove "-all" from the end of the JAR file name
+    archiveClassifier = null
+    // Add "v" to version to match previous releases
+    archiveVersion = "v" + project.version
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation(
+            project(':config'),
+            project(':reactclient'),
```

Comment:
Is that a typo or why would the client be required to compile the server part? (Or am I getting something wrong here)

Reply:
We must have this dependency here to allow build of react layer